src/styles/components/_navbar.scss

```diff
@@ -29,4 +21,46 @@
   @include respond-above('xl') {
     padding: 0 rem(40px);
   }
+
+  &__menu-item,
+  &__button--sign-in {
```

Comment:
FYI, you can actually just use `&__button`. No need for modifier since if there's `&__button--sign-in` and `&__button--sign-up` it will know who will be the last child.